includes/class-sensei-admin.php

```diff
@@ -1212,23 +1212,20 @@ public function get_course_order() {
 	}
 
 	public function save_course_order( $order_string = '' ) {
-		global $wpdb;
 		$order = array();
 
 		$i = 1;
 		foreach ( explode( ',', $order_string ) as $course_id ) {
 			if ( $course_id ) {
-				$order[] = $course_id;
-
-				// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching -- Performance improvement.
-				$wpdb->query(
-					$wpdb->prepare(
-						"UPDATE $wpdb->posts SET menu_order = %d WHERE ID = %d",
-						$i,
-						absint( $course_id )
-					)
+				$order[]     = $course_id;
+				$update_args = array(
+					'ID'         => absint( $course_id ),
+					'menu_order' => $i,
 				);
 
+				// If you face performance issues on Simple Sites, see https://github.com/Automattic/sensei/pull/7799.
```

Comment:
I added this comment, so in case we face this issue again in the future, it's easier to find our previous tries and possible solutions.